tests/testfiles/lunamark-markdown/auto-identifiers-with-attributes.test

```diff
@@ -0,0 +1,140 @@
+\def\markdownOptionHeaderAttributes{true}
+<<<
+This test ensures that the Lua `autoIdentifiers` option is enabled by default 
+with the Lua `headerAttributes` option set to true.
```

Comment:
This is inaccurate. The `autoIdentifiers` option was always enabled by default, regardless of the `headerAttributes` option. Since the two options don't interact, I am in favor of removing `auto-identifiers.test` and renaming `auto-identifiers-with-attributes.test` to just `auto-identifiers.test`.

Reply:
```suggestion
This test ensures that the Lua `autoIdentifiers` and `headerAttributes` options
correctly propagate through the plain TeX interface.
```

Reply:
I am in favor of removing `auto-identifiers.test` and renaming `auto-identifiers-with-attributes.test` to just `auto-identifiers.test`.